src/_locales/hu/messages.json

```diff
@@ -55,72 +55,72 @@
         "message": "Firefox for iOS"
     },
     "product_firefox-enterprise": {
-        "message": "Firefox for Enterprise"
+        "message": "Vállalati Firefox"
     },
     "congratulations": {
-        "message": "Congratulations! All questions have been answered!"
+        "message": "Gratulálunk! Az összes kérdés megválaszolva!"
     },
     "refresh": {
-        "message": "Refresh"
+        "message": "Frissítés"
     },
     "clear_notifications": {
-        "message": "Clear notifications"
+        "message": "Értesítések törlése"
     },
     "show_notifications": {
-        "message": "Show browser notifications"
+        "message": "Böngészőértesítések megjelenítése"
     },
     "description_notifications": {
         "message": "Determines whether you will receive notification popups when new questions are found"
```

Comment:
Can you take a look at that? I think you forgot that one. 

Thank you very much!  🙂

Reply:
Sure